Invalid Version: beta-0.0.0

### Expected Behavior
For discordjs-selfbot to install from GitHub as it looks like it's been removed from NPM

### Current Behavior
Throws `npm ERR! Invalid Version: beta-0.0.0` error

### Possible Solution
Using a SemVer compatible version in `package.json` and related files (https://semver.org/)

### Steps to Reproduce
1. Run `npm install https://github.com/TheDevYellowy/discordjs-selfbot`
2. Cry

### Context (Environment)
Node.js: v17.6.0
OS: Kubuntu 21.10 x86_64 - Linux Kernel 5.13.0-30-generic
Shell: bash 5.1.8

### Detailed Description
I believe the reason why NPM isn't working is because it requires a version number that is Semantic Versioning 2.0.0 compatible. The link I provided should be a good place to start.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "discordjs-selfbot",
-  "version": "beta-0.0.0",
+  "version": "0.0.1-beta",
   "description": "A unofficial discord.js fork for creating selfbots",
   "main": "./src/index.js",
   "types": "./typings/index.d.ts",

